src/datafactory/azext_datafactory/generated/_params.py

```diff
@@ -0,0 +1,633 @@
+        c.argument('if_match', help='ETag of the linkedService entity.  Should only be specified for update, for which '
+                   'it should match existing entity or can be * for unconditional update.')
+        c.argument('properties', arg_type=CLIArgumentType(options_list=['--properties'], help='Properties of linked ser'
+                                                          'vice. Expected value: json-string/@json-file.'))
+
+    with self.argument_context('datafactory linked-service update') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('factory_name', help='The factory name.', id_part='name')
+        c.argument('linked_service_name', options_list=['--name', '-n'], help='The linked service name.', id_part='chil'
+                   'd_name_1')
+        c.argument('if_match', help='ETag of the linkedService entity.  Should only be specified for update, for which '
+                   'it should match existing entity or can be * for unconditional update.')
+        c.argument('properties', arg_type=CLIArgumentType(options_list=['--properties'], help='Properties of linked ser'
+                                                          'vice. Expected value: json-string/@json-file.'))
+
+    with self.argument_context('datafactory linked-service delete') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('factory_name', help='The factory name.', id_part='name')
+        c.argument('linked_service_name', options_list=['--name', '-n'], help='The linked service name.', id_part='chil'
+                   'd_name_1')
+
+    with self.argument_context('datafactory dataset list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('factory_name', help='The factory name.')
+
+    with self.argument_context('datafactory dataset show') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('factory_name', help='The factory name.', id_part='name')
+        c.argument('dataset_name', options_list=['--name', '-n'], help='The dataset name.', id_part='child_name_1')
+        c.argument('if_none_match', help='ETag of the dataset entity. Should only be specified for get. If the ETag mat'
+                   'ches the existing entity tag, or if * was provided, then no content will be returned.')
+
+    with self.argument_context('datafactory dataset create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('factory_name', help='The factory name.')
+        c.argument('dataset_name', options_list=['--name', '-n'], help='The dataset name.')
+        c.argument('if_match', help='ETag of the dataset entity.  Should only be specified for update, for which it sho'
+                   'uld match existing entity or can be * for unconditional update.')

```

Comment:
Seeing line breaks inside a word multiple times. It's not an issue in the shown help message, but not good for code readability.

Reply:
This has not been fixed yet, the outdated status is outdated just because I changed the eof mark.

Reply:
This should've been fixed. Please check again, Thx